iceoryx2-ffi/cxx/include/iox2/event_id.hpp

```diff
@@ -13,33 +13,41 @@
 #ifndef IOX2_EVENT_ID_HPP
 #define IOX2_EVENT_ID_HPP
 
-#include "iox/assertions_addendum.hpp"
+#include "iox2/internal/iceoryx2.hpp"
 
 #include <cstdint>
 #include <iostream>
 
 namespace iox2 {
+/// Type that allows to identify an event uniquely.
 class EventId {
   public:
-    explicit EventId(const uint64_t value)
-        : m_value { value } {
-        IOX_TODO();
-    }
+    EventId(const EventId&) = default;
+    EventId(EventId&&) = default;
+    auto operator=(const EventId&) -> EventId& = default;
+    auto operator=(EventId&&) -> EventId& = default;
+    ~EventId() = default;
 
-    auto as_value() const -> uint64_t {
-        return m_value;
-    }
+    /// Creates a new uint128_t [`EventId`] from the high bit and low bit part.
+    explicit EventId(uint64_t low, uint64_t high = 0);
```

Comment:
Would you mind changing the order?
If one adds a number manually, e.g. `0x0...AFFE...0`, the current order would lead to `EventId(0xFE...0, 0x0...AF)`

There is a reasin big endian feels more natural :)

Reply:
I moved the high part to the end so that it can be optional, otherwise one has to always set the high part to zero if the user just requires "small" values.

Reply:
Obsolete now with usize